frontend/src/Layout/TopNavBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import {Navbar, Nav} from 'react-bootstrap';
+
+
+function TopNavBar(){
```

Comment:
`const TopNavBar = () : React.Node => (<> .... </>)` please do this for all the components